app/models.py

```diff
@@ -287,6 +292,16 @@ class User(db.Model, ModelMixin, UserMixin):
         db.Boolean, default=False, nullable=False, server_default="0"
     )
 
+    # whether to use random string or random word as suffix
```

Comment:
I think we need to create a migration file here, it can be created by running `new_migration.sh`

Reply:
I am unable to create that since I don't have the full installation of SL on my host computer, and it seems flask will not run in this case unless I manually find and install all missing packages

Reply:
Oh I see, I can run the script later when the PR is merged then.